app/views/projects/bots/_bot.html.erb

```diff
@@ -3,36 +3,36 @@
   class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
 >
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= bot_account.username.upcase %>
+    <%= bot_account.user.username.upcase %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= bot_account.personal_access_tokens.last.name %>
+    <%= bot_account.user.personal_access_tokens.last.name %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= bot_account.personal_access_tokens.last.scopes.join(", ") %>
+    <%= bot_account.user.personal_access_tokens.last.scopes.join(", ") %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= viral_time_ago(original_time: bot_account.created_at) %>
+    <%= viral_time_ago(original_time: bot_account.user.created_at) %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <% if bot_account.members.count.positive? %>
+    <% if bot_account.user.members.count.positive? %>
```

Comment:
**nitpick (blocking):** This should be changed to get the membership for the namespace specified in the namespace_bot entry.

Reply:
Updated in a37b856b3427dfd7bd949c10e4caa265bf956e47